src/html/partials/footer.html

```diff
@@ -0,0 +1,3 @@
+<footer class="page__footer">
```

Comment:
Similar BEM observation here, since there isn't a `page` class at the top, you don't necessarily have to have `page__footer` here, it could just be `page-footer` or just `footer` (for simplicity).